library/core/src/ops/arith.rs

```diff
@@ -194,6 +195,7 @@ pub trait Sub<Rhs = Self> {
     /// assert_eq!(12 - 1, 11);
     /// ```
     #[must_use = "this returns the result of the operation, without modifying the original"]
+    #[track_caller]
```

Comment:
Hmm, @bvanjoi, can you remind me why this is best on the trait here, rather than on the implementation that has the `#[rustc_inherit_overflow_checks]`?  (Like the one at line 212 below.)

It's not clear to me that we should force this on everyone's `Sub::sub`.

Reply:
As I recall, placing it here was simply a means to reduce some of the code

> Like the one at line 212 below.

Thank you for your reminder. I agree that it forms a better approach.